clients/admin-ui/src/features/auth/auth.slice.ts

```diff
@@ -81,8 +84,16 @@ export const authApi = createApi({
       }),
       invalidatesTags: () => ["Auth"],
     }),
+    getRolesToScopesMapping: build.query<RoleToScopes, void>({
```

Comment:
this isn't used yet, but was helpful in generating the cypress fixture. also it might get used in https://github.com/ethyca/fidesplus/issues/641